tests/test_flags/test_allow_empty.py

```
@@ -300,13 +301,12 @@ def test_allow_empty_with_no_verify(self):
                                                     git_commitai.main()
 
                                                     commit_calls = [
-                                                        call for call in mock_run.call_args_list
-                                                        if "commit" in str(call)
+                                                        c for c in mock_run.call_args_list
+                                                        if c.args and isinstance(c.args[0], list) and "commit" in c.args[0]
                                                     ]
                                                     if commit_calls:
                                                         last_cmd = commit_calls[-1].args[0]
```

Ruff (0.12.2)

Local variable `last_cmd` is assigned to but never used

Remove assignment to unused variable `last_cmd`

(F841)